app.js

```diff
@@ -22,23 +25,31 @@ app.get('/api/articles', sendArticles);
 
 app.get('/api/articles/:article_id', GetArticlesById);
 
+app.get('/api/articles/:article_id/comments', getCommentById)
 
+app.use(handleCustomErrors);
+app.use(handlePsqlErrors);
+app.use(handleServerErrors);
 
-app.use((err, req, res, next) => {
-  res.status(404).send({msg: 'Not Found'});
-});
 
-app.use((err, req, res, next) => {
-  if( err.code = '22P02' ){
-    res.status(400).send({msg: 'Invalid ID'})
-  }
-});
 
 
 
-app.use((err, req, res, next) => {
-  res.status(500).send('Server Error!');
-});
+// app.use((err, req, res, next) => {
+//   if( err.code = '22P02' ){
+//   res.status(404).send({msg: 'Not Found'});
+//   }else next(err);
+// });
+
```

Comment:
Commented out code and console logs shouldn't be in pull requests, this is finished code so we shouldnt need console logs or anything thats commented out.